### PR TITLE
limit user input for post and comment

### LINKED ISSUE #65: Limit user input on PostForm (1)
Technical Background:
- if user input is too long for `content`, an error is returned. We should limit user input before they submit a request to backend API.
- use [this code snippet](https://codesandbox.io/s/material-ui-text-field-character-limit-n4t82) as a reference

Acceptance Criteria:
- Showing **word limit** on `PostForm` to let users know the number of words they are allowed to input.

### PR DESCRIPTION
**Pull request by nchen000**
### Issues
closes #65 

### Overview of PR

- [x] limit user input so no error is returned by backend API

### UI change
- post 
image
- comment
image

### CODE REVIEW

File: src/components/Forms/CommentInput/CommentInput.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ const preparePayload = (method, data) => {
 };
 
 export const CommentInput = (props) => {
+  const INPUT_CHAR_LIMIT = 1000;
```
Comment (songrenzhao) [inline src/components/Forms/CommentInput/CommentInput.js]: Should be 2000, our database model restricts the input size to be 2000.

**Comment by nchen000** [comment]
> Is the size limit match the backend database limitation?

Yes. I double checked the models in Backend repo to make sure the limitations are correct.

File: src/components/Forms/CommentInput/CommentInput.js (hunk 1 above)
Comment (nchen000) [reply src/components/Forms/CommentInput/CommentInput.js]: The model for [comment (line 23) ](https://github.com/jinchen1036/rdb-crud-microservice/blob/develop/src/models/comments.js) states the limit is 1000.

Comment (songrenzhao) [reply src/components/Forms/CommentInput/CommentInput.js]: Got it, it's closed

**Pull request merged**